Readme.md

```diff
@@ -17,14 +17,16 @@
 [![Feature Requests](https://img.shields.io/github/issues/aws-amplify/amplify-cli/feature-request?color=ff9001&label=feature%20requests)](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+label%3Afeature-request+is%3Aopen)
 [![Closed Issues](https://img.shields.io/github/issues-closed/aws-amplify/amplify-cli?color=%2325CC00&label=issues%20closed)](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+is%3Aclosed+)
 
-> [!TIP]  
-> **Preview: AWS Amplify's new code-first DX (Gen 2) for building backends**
+> [!Important]  
+> # Amplify Gen 2 is now available
 >
-> The next generation of Amplify's backend building experience with a TypeScript-first DX is now available in Preview. To learn more, please visit [AWS Amplify (Gen 2)](https://docs.amplify.aws/gen2/).
+> If you are starting a new project, we recommend starting with [Amplify Gen 2](https://github.com/aws-amplify/amplify-backend).
 >
-> To open a new issue related to Amplify's new backend building experience, please visit [AWS Amplify Backend](https://github.com/aws-amplify/amplify-backend).
+> For Gen 1 customers, we are actively developing migration tooling to aid in transitioning your project from Gen 1 to Gen 2. Until then, we recommend you continue working with your Gen 1 Amplify project. We remain committed to supporting both Gen 1 and Gen 2 for the foreseeable future. For new projects, we recommend adopting Gen 2 to take advantage of its enhanced capabilities.
```

Comment:
If you are an existing Gen 1 customer, we recommend that you continue working with your Gen 1 Amplify project. We are actively developing migration tooling to aid in transitioning your project from Gen 1 to Gen 2.  We remain committed to supporting both Gen 1 and Gen 2 for the foreseeable future.